logic/style_transfer.py

```diff
@@ -63,12 +63,13 @@ def stylize_image(content_image, style_image, content_blending_ratio):
         return result_image
 
     @staticmethod
-    def stylize_video(content_video_path, style_image_path, content_blending_ratio):
+    def stylize_video(content_video_path, style_image_path, content_blending_ratio, progress_signal):
         style_image = preprocess_image(load_img(style_image_path), 256)
         video_capture_object = cv.VideoCapture(content_video_path)
 
         frame_size = (384, 384)
         frame_rate = video_capture_object.get(cv.CAP_PROP_FPS)
+        frames_number = int(video_capture_object.get(cv.CAP_PROP_FRAME_COUNT))
         out = cv.VideoWriter("assets/results/result_video.avi", cv.VideoWriter_fourcc(*'DIVX'), frame_rate, frame_size)
 
         count = 0
```

Comment:
ta zmienna jest dość kluczowa i moim zdaniem powinna nazywać się bardziej specyficznie, np. `frame_counter`

Reply:
jedna mała zmiana a tak to dobra robota